corehq/apps/domain/tests/test_forms.py

```diff
@@ -1,25 +1,17 @@
-from django.test import SimpleTestCase, TestCase
+from django.test import SimpleTestCase
 from unittest.mock import Mock, patch
 from corehq.apps.domain.models import Domain, OperatorCallLimitSettings
+from corehq import privileges
 
-from corehq.toggles import NAMESPACE_DOMAIN, TWO_STAGE_USER_PROVISIONING_BY_SMS
-from corehq.toggles.shortcuts import set_toggle
-
-from ..forms import DomainGlobalSettingsForm, PrivacySecurityForm
+from ..forms import DomainGlobalSettingsForm, DomainMetadataForm, PrivacySecurityForm
 from .. import forms
 
 
 class PrivacySecurityFormTests(SimpleTestCase):
```

Comment:
I realize this test was named in an earlier PR, but most of our test class names start with `Test` (not end with `Tests`). Is that a convention you're hoping to change or is this consistent with other tests in this area?

Reply:
That's a convention I was hoping to change, but I guess its not a big deal.